src/main/java/org/broadinstitute/hellbender/tools/gvs/ingest/PetCreator.java

```diff
@@ -399,9 +437,18 @@ public static GQStateEnum getGQStateEnum(int GQ){
         return Arrays.stream(PetFieldEnum.values()).map(String::valueOf).collect(Collectors.toList());
     }
 
+    public void completeCreation() {
+        if (outputType == CommonCode.OutputType.BQ && petBQJsonWriter != null) {
+            petBQJsonWriter.commitWriteStreams();
```

Comment:
Is there any way to have both these streams come from the same place to they get committed together?  No idea if that's a thing, but right now we could commit one and then fail on the other

Reply:
yeah, I really wanted to be able to do that, but the bqclient is per table so I couldn't create one that streamed to both tables. I'll ask in the google meeting.